modules/test/jenkins-results-parser/src/main/java/com/liferay/jenkins/results/parser/BatchTestClassGroup.java

```diff
@@ -57,13 +61,105 @@ protected BatchTestClassGroup(
 			new File(
 				this.gitWorkingDirectory.getWorkingDirectory(),
 				"test.properties"));
+
+		_setTestRelevantChanges();
+	}
+
+	protected String getFirstPropertyValue(
+		Properties properties, List<String> propertyNames) {
+
+		for (String propertyName : propertyNames) {
+			if (propertyName == null) {
+				continue;
+			}
+
+			if (properties.containsKey(propertyName)) {
+				String propertyValue = properties.getProperty(propertyName);
+
+				if ((propertyValue != null) && !propertyValue.isEmpty()) {
+					return propertyValue;
+				}
+			}
+		}
+
+		return null;
+	}
+
+	protected String getWildcardPropertyName(
+		Properties properties, String propertyName) {
+
+		return getWildcardPropertyName(properties, propertyName, null);
+	}
+
+	protected String getWildcardPropertyName(
+		Properties properties, String propertyName, String testSuiteName) {
+
+		for (String wildcardPropertyName : properties.stringPropertyNames()) {
+			if (!wildcardPropertyName.startsWith(propertyName)) {
+				continue;
+			}
+
+			Matcher matcher = _propertyNamePattern.matcher(
+				wildcardPropertyName);
+
+			if (matcher.find()) {
+				String batchNameMatcher = matcher.group("batchName");
+
+				batchNameMatcher = batchNameMatcher.replace("*", ".+");
+
+				if (!batchName.matches(batchNameMatcher)) {
+					continue;
+				}
+
+				String testSuiteNameMatcher = matcher.group("testSuiteName");
+
+				if ((testSuiteName != null) &&
+					!testSuiteName.equals(testSuiteNameMatcher)) {
+
+					continue;
+				}
+
```

Comment:
This should have been caught when this code was first written but you we shouldn't be calling variables `xxxMatcher` when they are not actually `Matcher` instances. It looks more like we are creating a regex here (for `batchNameMatcher`). If that's the case we should call it that.

For `testSuiteNameMatcher`, you could still call it a regex if you used 

`!testSuiteName.matches(testSuiteNameRegex)`

instead of

`!testSuiteName.equals(testSuiteNameMatcher)`

Alternatively, you don't necessarily need to create a variable at all and instead refer directly to the `matcher.group("testSuiteName"))` result.

`!testSuiteName.equals(matcher.group("testSuiteName")`